cbmc/proofs/Makefile.common

```diff
@@ -51,15 +51,35 @@ CFLAGS += $(CFLAGS2) $(INC) $(DEF) -std=gnu99
 
 NONDET += --nondet-static
 
-%.goto : %.c
+# Out of source builds
+#
+# Proof makefiles were originally written using in-source builds, but
+# this inhibits building and running proofs concurrently when
+# different proofs build sources with different configurations.  These
+# definitions assume $(OBJS) is the original list of in-source goto
+# binaries and defines $(GOTOS) to be the list of out-of-source goto
+# binaries written into the local gotos subdirectory.
+
+SRCS=$(patsubst %.goto,%.c,$(patsubst $(MQTT)/%,%,$(OBJS)))
+GOTO_DIR=gotos
+GOTOS=$(patsubst %.c,$(GOTO_DIR)/%.goto,$(SRCS))
+
+# Build sources in the source tree with full paths
+$(GOTO_DIR)/%.goto:$(MQTT)/%.c
```

Comment:
Having a space after the colon is easier to read

Reply:
Done.